aws-logging.yaml

```diff
@@ -0,0 +1,44 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  # NEVER set this to true, or set only temporary
+  # it enables internal fluentbit logs that flood Cloudwatch
+  # and cause huge AWS bills 
+  # (see slack post https://this-is-biomage.slack.com/archives/C014YMUT6GN/p1646915799716329)
+  flb_log_cw: "false"
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch
+        Match *
+        region eu-west-1
+        log_group_name fluent-bit-cloudwatch
+        log_stream_name $(kubernetes['pod_name'])
+        auto_create_group true
+
+  parsers.conf: |
+    [PARSER]
+        Name         docker
+        Format       json
+        Time_Key     time
+        Time_Format  %Y-%m-%dT%H:%M:%S.%L
+        Time_Keep    On
+  
+  filters.conf: |
+    [FILTER]
```

Comment:
I didn't try this, but it looks useful for adding some more metadata to the logs: https://docs.fluentbit.io/manual/pipeline/filters/aws-metadata